### PR TITLE
2020 fall imr changes

### PR DESCRIPTION
**Pull request by midran**
The problem with the build seems to have been resolved. The IMR pull request does not include an update to the release notes because the branch is taken directly from the release candidate branch and the section file for the fall release is not there. Its simpler to update the release notes after merging the existing branches rather than creating a conflicting file.

### CODE REVIEW

File: rules/examples/Invoice reponse use cases/T111-uc007-Payment has been initiated.xml
Comment (jerouris) [inline rules/examples/Invoice reponse use cases/T111-uc007-Payment has been initiated.xml:43]: This Addition to the example does not validate against the IMR Schematron. Consider moving the DocumentReference Inside the DocumentResponse or fix the syntax rules of IMR

File: guides/profiles/63-invoiceresponse/process/use-cases/use-cases.adoc
Comment (jerouris) [inline guides/profiles/63-invoiceresponse/process/use-cases/use-cases.adoc:220]: Status PAID is code PD? This should be clearly stated and tested against. (Eg. is it allowed to have PPD Clarification Reason with status code other than PD?)

File: rules/examples/Invoice reponse use cases/T111-uc009-Partial payment has been initiated.xml
Comment (jerouris) [inline rules/examples/Invoice reponse use cases/T111-uc009-Partial payment has been initiated.xml:35]: The Response Code appears to be wrong, but the Schematron seem to allow it. Either fix the specification or fix the schematrons

File: rules/sch/parts/PEPPOL-M-T111.sch
Comment (jerouris) [inline rules/sch/parts/PEPPOL-M-T111.sch:22]: There is no rule that checks the affinity of codes (PPD only applicable when PD is Response Code). If response codes other than PD are allowed this should be clearly stated in the specification document, otherwise, the schematron rules need to be further updated.

**Review by jerouris — CHANGES_REQUESTED** [review]
There are issues that need to be fixed, please see inline comments to files.

File: rules/examples/Invoice reponse use cases/T111-uc009-Partial payment has been initiated.xml
Comment (midran) [reply rules/examples/Invoice reponse use cases/T111-uc009-Partial payment has been initiated.xml:35]: Changed as proposed

File: guides/profiles/63-invoiceresponse/process/use-cases/use-cases.adoc
Comment (midran) [reply guides/profiles/63-invoiceresponse/process/use-cases/use-cases.adoc:220]: Changed as proposed

File: rules/examples/Invoice reponse use cases/T111-uc007-Payment has been initiated.xml
Comment (midran) [reply rules/examples/Invoice reponse use cases/T111-uc007-Payment has been initiated.xml:43]: I was trying to improve the clarity of the documentation by showing the two business terms Invoice id and Payment id, separately instead of combining them into the same element. That does not seem to work because Travis builds the syntax validation rules from this and when the same element is shown twice it does not seem to work out. The issue is that the syntax is 1..1, default but when adding the option to give payment reference the cardinality needs to be extened. Since the intention is to support two identifiers and not unbounded and one of those identifiers, the reference to the invoice, as mandatory and the payment reference is optional up to once the suitable cardinality. To solve this I removed the dublicate document referencd syntax and change the cardinality of the existing one to 1..2. The description and business term identifier are then merged. This however means that additional rules are needed to make sure that the invoice reference is included at least once otherwise it would be possible to create a message with payment reference only. Suggest however to merge this change as is, if it works, and add the rules in a new branch.

File: rules/sch/parts/PEPPOL-M-T111.sch
Comment (midran) [reply rules/sch/parts/PEPPOL-M-T111.sch:22]: QA shows that following additional rules are needed.

1. Check that PPD can only be used along with PD.
2. Check that document reference is included at least once using document type code 38

Propose to add these in a separate branch and merge this branch if other parts are OK.

**Comment by jerouris** [comment]
Merging with the two issues pending, with the promise to be fixed before the merge of the RC to Master and publication

**Review by jerouris — APPROVED** [review]
Approved but with the promise of fixing the pending rules before the RC is merged with Master

**Pull request merged**
